src/VisualLinter/Helpers/StylelintHelper.cs

```diff
@@ -0,0 +1,292 @@
+
+                OutputWindowHelper.DebugLine($"using global stylelint @ {globalPath}");
+
+                return globalPath;
+            }
+
+            // resolve local stylelint path
+            var localPath = GetLocalStylelintPath(directoryPath) ??
+                throw new Exception("exception: no local stylelint found-- is stylelint installed locally?");
+
+            OutputWindowHelper.DebugLine($"using local stylelint @ {localPath}");
+
+            return localPath;
+        }
+
+        internal static string GetIgnorePath(string directoryPath)
+        {
+            OutputWindowHelper.DebugLine($"DisableStylelintIgnore: {Options.DisableStylelintIgnore}");
+
+            // disable stylelint ignore
+            if (Options.DisableStylelintIgnore)
+            {
+                OutputWindowHelper.DebugLine("not using stylelint ignore");
+
+                return null;
+            }
+
+            OutputWindowHelper.DebugLine($"ShouldOverrideStylelintIgnorePath: {Options.ShouldOverrideStylelintIgnorePath}");
+
+            // resolve override stylelint ignore path
+            if (Options.ShouldOverrideStylelintIgnorePath)
+            {
+                var overridePath = GetOverrideStylelintIgnore() ??
+                    throw new Exception("exception: option 'Override Stylelint ignore path' is set to true-- but no path is set");
+
+                OutputWindowHelper.DebugLine($"using override stylelint ignore @ {overridePath}");
+
+                return ValidateOverridePath(overridePath);
+            }
+
+            // resolve stylelint ignore path
+            var solutionPath = VsixHelper.GetSolutionPath() ??
+                throw new Exception("exception: could not get solution path");
+
+            var workingDirectory = new DirectoryInfo(directoryPath);
+
+            var path = FindRecursive(workingDirectory, solutionPath, ResolveIgnorePath);
+
+            OutputWindowHelper.DebugLine($"using stylelint ignore @ {path ?? "not found"}");
+
+            return path;
+        }
+
+        private static string FindRecursive(DirectoryInfo workingDirectory, string end, Func<DirectoryInfo, string> fn)
+        {
+            do
+            {
+                var found = fn(workingDirectory);
+
+                if (null != found)
+                    return found;
+
+                workingDirectory = workingDirectory.Parent;
+
+                if (null == workingDirectory)
+                    return null;
+            } while (-1 != workingDirectory.FullName.IndexOf(end, StringComparison.OrdinalIgnoreCase));
+
+            return null;
+        }
+
+        private static string FormatArguments(IReadOnlyDictionary<string, string> arguments)
+        {
+            return string.Join(" ", arguments.Select(arg => $"--{arg.Key}=\"{arg.Value}\""));
+        }
+
+        private static string GetGlobalStylelintPath()
```

Comment:
This should be modified to accept a `string variableName` parameter and be a shared method between `EslintHelper.cs` and `StylelintHelper.cs`.